R/population-workflow.R

```diff
@@ -288,15 +274,15 @@ PopulationWorkflow <- R6::R6Class(
 
           set$sensitivityAnalysisResultsFileNames <- runPopulationSensitivityAnalysis(
             simFilePath = set$simulationSet$simulationFile,
-            variableParameterPaths = self$populationSensitivityAnalysis$variableParameterPaths,
-            popDataFilePath = set$simulationSet$populationFile, # file.path(set$inputFilesFolder, paste0(set$simulationSet$populationName, ".csv")),
+            popDataFilePath = set$simulationSet$populationFile,
             pkParameterResultsFilePath = set$pkAnalysisResultsFileNames,
-            pkParameterSelection = self$populationSensitivityAnalysis$pkParameterSelection,
             resultsFileFolder = file.path(self$populationSensitivityAnalysis$workflowFolder, self$populationSensitivityAnalysis$outputFolder),
             resultsFileName = trimFileName(defaultFileNames$sensitivityAnalysisResultsFile(set$simulationSet$simulationSetName), extension = "csv"),
-            variationRange = self$populationSensitivityAnalysis$variationRange,
-            quantileVec = self$populationSensitivityAnalysis$quantileVec,
-            numberOfCores = self$populationSensitivityAnalysis$numberOfCores,
+            variableParameterPaths = self$populationSensitivityAnalysis$settings$variableParameterPaths,

```

Comment:
same here. The settings should be a parameter of the function. Why do you spread all parameters if you have an object?

Reply:
This will be done soon, this PR was just to get the examples working again after the last change.